Solutions/Box/Analytic Rules/BoxUserLoginAsAdmin.yaml

```diff
@@ -26,16 +26,14 @@ query: |
   BoxEvents
   | where EventType =~ 'ADMIN_LOGIN'
   | where SourceLogin !in (admins)
-  | extend AccountCustomEntity = SourceLogin
-  | extend IPCustomEntity = SrcIpAddr
 entityMappings:

```

Comment:
same as above. Add Upndomain as strong identifiers. Also add output formatting and aggregations.